unittests/emitc_tosa.cpp

```diff
@@ -31,4 +34,43 @@ TEST(tosa, reciprocal) {
   EXPECT_THAT(result, Pointwise(FloatNear(EPSILON), excpected));
 }
 
+// Binary ops
```

Comment:
```suggestion
// Binary elementwise ops
```
To be consistent with your change above :)